cross_country_aggregate_dlt.py

```diff
@@ -25,6 +25,7 @@
     StructField("executed", DoubleType(), True, {'comment': 'Actual amount spent in current local currency by the end of the fiscal year.'})
 ])
 
+@dlt.expect_or_drop("executed not null nor 0", "executed IS NOT NULL and executed != 0")
```

Comment:
Update it to drop if both executed and approved are null or 0

Reply:
done!